packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -35,31 +39,34 @@ export function treeItemTemplate(
                 filter: elements(),
             })}
         >
-            <div class="positioning-region" part="positioning-region">
-                <div class="content-region" part="content-region">
-                    ${when(
-                        x => x.childItems && x.childItemLength(),
-                        html<FASTTreeItem>`
-                            <div
-                                aria-hidden="true"
-                                class="expand-collapse-button"
-                                part="expand-collapse-button"
-                                @click="${(x, c) =>
-                                    x.handleExpandCollapseButtonClick(
-                                        c.event as MouseEvent
-                                    )}"
-                                ${ref("expandCollapseButton")}
+            <div class="treeitem" part="treeitem">
```

Comment:
The entire thing (the host) is a tree-item, I don't know that this creates more clarity.